cmd/api/src/daemons/datapipe/jobs.go

```diff
@@ -151,6 +151,11 @@ func (s *Daemon) processIngestTasks(ctx context.Context, ingestTasks model.Inges
 			return
 		}
 
+		if s.cfg.DisableIngest {
```

Comment:
I'm looking for specific feedback on the placement of this check. I originally had it during the startup process and would just not start the data pipe daemon if the flag was set to `true`. One thing I didn't like about this implementation is that it wasn't clear why a file upload might not get past the ingesting step if the flag was unknowingly enabled. The benefit of setting it here is it will spam the logs with this message if `ingest_tasks` exist but the flag is enabled.

Reply:
This seems like a perfectly reasonable compromise. This will only be an option devs will enable (it's not meant to be user consumable as there's no practical reason we know of for a user to want this behavior) so having some logging to remind the dev that ingest is disabled will prevent unnecessary digging for a non-existent bug